src/app/components/filtering-criteria-block/filtering-criteria-block.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="header__container-sort">
+    <span>Sorting by:</span>
+    <div class="sort">
+        <a
+            href="#"
+            (click)="sortService.sortByDate($event); setActiveSorting('date')"
```

Comment:
it is better to create a separate method in component fot click event

Reply:
```
<a
            href="#"
            (click)="handleSortByDate($event)"
            [ngClass]="getLinkClass('date')"
        >
            date
            <mat-icon *ngIf="sortService.sortBy === 'date'">
                {{
                    sortService.sortOrder === "asc"
                        ? "arrow_upward"
                        : "arrow_downward"
                }}
            </mat-icon>
        </a>
```
```
handleSortByDate(event: Event) {
        this.sortService.sortByDate(event);
        this.setActiveSorting('date');
    }
```

Fixed!